2024-08-06 20:01:58,590 INFO    StreamThr :2148567 [internal.py:wandb_internal():86] W&B internal server running at pid: 2148567, started at: 2024-08-06 20:01:58.590250
2024-08-06 20:01:58,591 DEBUG   HandlerThread:2148567 [handler.py:handle_request():146] handle_request: status
2024-08-06 20:01:58,592 INFO    WriterThread:2148567 [datastore.py:open_for_write():85] open: /home/rwkv/JL/GMM/llama/wandb/run-20240806_200158-zo7etxt6/run-zo7etxt6.wandb
2024-08-06 20:01:58,592 DEBUG   SenderThread:2148567 [sender.py:send():382] send: header
2024-08-06 20:01:58,592 DEBUG   SenderThread:2148567 [sender.py:send():382] send: run
2024-08-06 20:01:58,764 INFO    SenderThread:2148567 [dir_watcher.py:__init__():211] watching files in: /home/rwkv/JL/GMM/llama/wandb/run-20240806_200158-zo7etxt6/files
2024-08-06 20:01:58,764 INFO    SenderThread:2148567 [sender.py:_start_run_threads():1136] run started: zo7etxt6 with start time 1722945718.590173
2024-08-06 20:01:58,784 DEBUG   HandlerThread:2148567 [handler.py:handle_request():146] handle_request: check_version
2024-08-06 20:01:58,784 DEBUG   SenderThread:2148567 [sender.py:send_request():409] send_request: check_version
2024-08-06 20:01:58,906 DEBUG   HandlerThread:2148567 [handler.py:handle_request():146] handle_request: run_start
2024-08-06 20:01:58,915 DEBUG   HandlerThread:2148567 [system_info.py:__init__():32] System info init
2024-08-06 20:01:58,916 DEBUG   HandlerThread:2148567 [system_info.py:__init__():47] System info init done
2024-08-06 20:01:58,916 INFO    HandlerThread:2148567 [system_monitor.py:start():194] Starting system monitor
2024-08-06 20:01:58,916 INFO    SystemMonitor:2148567 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-08-06 20:01:58,916 INFO    HandlerThread:2148567 [system_monitor.py:probe():214] Collecting system info
2024-08-06 20:01:58,916 INFO    SystemMonitor:2148567 [interfaces.py:start():190] Started cpu monitoring
2024-08-06 20:01:58,916 INFO    SystemMonitor:2148567 [interfaces.py:start():190] Started disk monitoring
2024-08-06 20:01:58,917 INFO    SystemMonitor:2148567 [interfaces.py:start():190] Started gpu monitoring
2024-08-06 20:01:58,917 INFO    SystemMonitor:2148567 [interfaces.py:start():190] Started memory monitoring
2024-08-06 20:01:58,917 INFO    SystemMonitor:2148567 [interfaces.py:start():190] Started network monitoring
2024-08-06 20:01:58,922 DEBUG   HandlerThread:2148567 [system_info.py:probe():196] Probing system
2024-08-06 20:01:58,924 DEBUG   HandlerThread:2148567 [system_info.py:_probe_git():181] Probing git
2024-08-06 20:01:58,925 DEBUG   HandlerThread:2148567 [system_info.py:_probe_git():189] Probing git done
2024-08-06 20:01:58,925 DEBUG   HandlerThread:2148567 [system_info.py:probe():244] Probing system done
2024-08-06 20:01:58,925 DEBUG   HandlerThread:2148567 [system_monitor.py:probe():223] {'os': 'Linux-6.5.0-41-generic-x86_64-with-glibc2.35', 'python': '3.10.13', 'heartbeatAt': '2024-08-06T12:01:58.922861', 'startedAt': '2024-08-06T12:01:58.585231', 'docker': None, 'cuda': None, 'args': ('--local_rank=0', '--deepspeed', 'configs/ds_config_zero2_no_offload.json', '--model_name_or_path', '/home/rwkv/JL/model/qwen2-1.5b', '--data_path', '/home/rwkv/JL/model/MetaMathQA', '--dataset_field', 'query', 'response', '--dataset_split', 'train[:395000]', '--output_dir', '/home/rwkv/JL/out_model/Gemma-meta-2btest', '--num_train_epochs', '1', '--model_max_length', '512', '--per_device_train_batch_size', '2', '--gradient_accumulation_steps', '16', '--save_strategy', 'steps', '--save_steps', '1000', '--save_total_limit', '2', '--learning_rate', '1e-5', '--weight_decay', '0.', '--warmup_ratio', '0.', '--logging_steps', '1', '--lr_scheduler_type', 'cosine', '--report_to', 'wandb', '--merge', 'True'), 'state': 'running', 'program': '/home/rwkv/JL/GMM/llama/gmm.py', 'codePathLocal': 'gmm.py', 'codePath': 'llama/gmm.py', 'git': {'remote': 'https://github.com/JL-er/GMM.git', 'commit': None}, 'email': '997529190@qq.com', 'root': '/home/rwkv/JL/GMM', 'host': 'rwkv-Z790M-AORUS-ELITE-AX', 'username': 'rwkv', 'executable': '/home/rwkv/miniconda3/bin/python', 'cpu_count': 20, 'cpu_count_logical': 28, 'cpu_freq': {'current': 2265.6622142857145, 'min': 800.0, 'max': 5000.0}, 'cpu_freq_per_core': [{'current': 5500.0, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 5500.0, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 5390.424, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 5500.0, 'min': 800.0, 'max': 5500.0}, {'current': 1342.924, 'min': 800.0, 'max': 5500.0}, {'current': 5500.0, 'min': 800.0, 'max': 5600.0}, {'current': 800.0, 'min': 800.0, 'max': 5600.0}, {'current': 5500.0, 'min': 800.0, 'max': 5600.0}, {'current': 800.0, 'min': 800.0, 'max': 5600.0}, {'current': 2694.029, 'min': 800.0, 'max': 5500.0}, {'current': 2821.01, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 5500.0, 'min': 800.0, 'max': 5500.0}, {'current': 4290.337, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 1099.818, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}], 'disk': {'/': {'total': 916.2753829956055, 'used': 443.3793487548828}}, 'gpu': 'NVIDIA GeForce RTX 4090', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 4090', 'memory_total': 25757220864}], 'memory': {'total': 62.5532341003418}}
2024-08-06 20:01:58,925 INFO    HandlerThread:2148567 [system_monitor.py:probe():224] Finished collecting system info
2024-08-06 20:01:58,925 INFO    HandlerThread:2148567 [system_monitor.py:probe():227] Publishing system info
2024-08-06 20:01:58,925 DEBUG   HandlerThread:2148567 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2024-08-06 20:01:58,926 DEBUG   HandlerThread:2148567 [system_info.py:_save_pip():68] Saving pip packages done
2024-08-06 20:01:58,926 DEBUG   HandlerThread:2148567 [system_info.py:_save_conda():75] Saving list of conda packages installed into the current environment
2024-08-06 20:01:59,764 INFO    Thread-12 :2148567 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_200158-zo7etxt6/files/requirements.txt
2024-08-06 20:01:59,764 INFO    Thread-12 :2148567 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_200158-zo7etxt6/files/conda-environment.yaml
2024-08-06 20:02:00,586 DEBUG   HandlerThread:2148567 [system_info.py:_save_conda():87] Saving conda packages done
2024-08-06 20:02:00,586 DEBUG   HandlerThread:2148567 [system_info.py:_save_code():90] Saving code
2024-08-06 20:02:00,587 DEBUG   HandlerThread:2148567 [system_info.py:_save_code():111] Saving code done
2024-08-06 20:02:00,587 DEBUG   HandlerThread:2148567 [system_info.py:_save_patches():128] Saving git patches
2024-08-06 20:02:00,590 ERROR   HandlerThread:2148567 [system_info.py:_save_patches():169] Error generating diff: Reference at 'refs/remotes/origin/main' does not exist
2024-08-06 20:02:00,590 DEBUG   HandlerThread:2148567 [system_info.py:_save_patches():170] Saving git patches done
2024-08-06 20:02:00,590 INFO    HandlerThread:2148567 [system_monitor.py:probe():229] Finished publishing system info
2024-08-06 20:02:00,591 DEBUG   SenderThread:2148567 [sender.py:send():382] send: files
2024-08-06 20:02:00,591 INFO    SenderThread:2148567 [sender.py:_save_file():1392] saving file wandb-metadata.json with policy now
2024-08-06 20:02:00,591 INFO    SenderThread:2148567 [sender.py:_save_file():1392] saving file code/llama/gmm.py with policy now
2024-08-06 20:02:00,594 DEBUG   HandlerThread:2148567 [handler.py:handle_request():146] handle_request: stop_status
2024-08-06 20:02:00,594 DEBUG   SenderThread:2148567 [sender.py:send_request():409] send_request: stop_status
2024-08-06 20:02:00,634 DEBUG   HandlerThread:2148567 [handler.py:handle_request():146] handle_request: internal_messages
2024-08-06 20:02:00,764 INFO    Thread-12 :2148567 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_200158-zo7etxt6/files/conda-environment.yaml
2024-08-06 20:02:00,764 INFO    Thread-12 :2148567 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_200158-zo7etxt6/files/code/llama/gmm.py
2024-08-06 20:02:00,764 INFO    Thread-12 :2148567 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_200158-zo7etxt6/files/wandb-metadata.json
2024-08-06 20:02:00,764 INFO    Thread-12 :2148567 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_200158-zo7etxt6/files/code/llama
2024-08-06 20:02:00,764 INFO    Thread-12 :2148567 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_200158-zo7etxt6/files/code
2024-08-06 20:02:00,797 DEBUG   SenderThread:2148567 [sender.py:send():382] send: telemetry
2024-08-06 20:02:00,797 DEBUG   SenderThread:2148567 [sender.py:send():382] send: config
2024-08-06 20:02:00,797 DEBUG   SenderThread:2148567 [sender.py:send():382] send: metric
2024-08-06 20:02:00,797 DEBUG   SenderThread:2148567 [sender.py:send():382] send: telemetry
2024-08-06 20:02:00,797 DEBUG   SenderThread:2148567 [sender.py:send():382] send: metric
2024-08-06 20:02:00,797 WARNING SenderThread:2148567 [sender.py:send_metric():1343] Seen metric with glob (shouldn't happen)
2024-08-06 20:02:00,797 DEBUG   SenderThread:2148567 [sender.py:send():382] send: telemetry
2024-08-06 20:02:00,797 DEBUG   SenderThread:2148567 [sender.py:send():382] send: telemetry
2024-08-06 20:02:00,797 DEBUG   SenderThread:2148567 [sender.py:send():382] send: config
2024-08-06 20:02:00,849 ERROR   wandb-upload_0:2148567 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/zo7etxt6/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T120200Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=8770319683b94cf3b7889fb807301df06cdc0e102e6e00518805bd314884a1aa: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/zo7etxt6/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T120200Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=8770319683b94cf3b7889fb807301df06cdc0e102e6e00518805bd314884a1aa
2024-08-06 20:02:00,849 ERROR   wandb-upload_0:2148567 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5174', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 20:02:00,849 ERROR   wandb-upload_0:2148567 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/zo7etxt6/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/zo7etxt6/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E92201292E20B6</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 20:02:00,992 ERROR   wandb-upload_1:2148567 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/zo7etxt6/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T120200Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=50a85cf9e52b42c40f2c8d915d09a31c5e68476df98b8fa9ee6a4cd2ac8adce4: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/zo7etxt6/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T120200Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=50a85cf9e52b42c40f2c8d915d09a31c5e68476df98b8fa9ee6a4cd2ac8adce4
2024-08-06 20:02:00,992 ERROR   wandb-upload_1:2148567 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 20:02:00,992 ERROR   wandb-upload_1:2148567 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/zo7etxt6/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/zo7etxt6/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E9220130DEAB2A</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 20:02:01,764 INFO    Thread-12 :2148567 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_200158-zo7etxt6/files/output.log
2024-08-06 20:02:02,178 ERROR   wandb-upload_0:2148567 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/zo7etxt6/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T120200Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=8770319683b94cf3b7889fb807301df06cdc0e102e6e00518805bd314884a1aa: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/zo7etxt6/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T120200Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=8770319683b94cf3b7889fb807301df06cdc0e102e6e00518805bd314884a1aa
2024-08-06 20:02:02,178 ERROR   wandb-upload_0:2148567 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5174', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 20:02:02,178 ERROR   wandb-upload_0:2148567 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/zo7etxt6/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/zo7etxt6/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E92201781A0B4D</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 20:02:02,305 ERROR   wandb-upload_1:2148567 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/zo7etxt6/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T120200Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=50a85cf9e52b42c40f2c8d915d09a31c5e68476df98b8fa9ee6a4cd2ac8adce4: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/zo7etxt6/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T120200Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=50a85cf9e52b42c40f2c8d915d09a31c5e68476df98b8fa9ee6a4cd2ac8adce4
2024-08-06 20:02:02,305 ERROR   wandb-upload_1:2148567 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 20:02:02,305 ERROR   wandb-upload_1:2148567 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/zo7etxt6/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/zo7etxt6/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E922017EC85894</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 20:02:02,305 INFO    wandb-upload_1:2148567 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/zo7etxt6/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T120200Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=50a85cf9e52b42c40f2c8d915d09a31c5e68476df98b8fa9ee6a4cd2ac8adce4

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-06 20:02:02,534 DEBUG   HandlerThread:2148567 [handler.py:handle_request():146] handle_request: partial_history
2024-08-06 20:02:02,535 DEBUG   SenderThread:2148567 [sender.py:send():382] send: metric
2024-08-06 20:02:02,535 DEBUG   SenderThread:2148567 [sender.py:send():382] send: metric
2024-08-06 20:02:02,535 DEBUG   SenderThread:2148567 [sender.py:send():382] send: metric
2024-08-06 20:02:02,535 DEBUG   SenderThread:2148567 [sender.py:send():382] send: metric
2024-08-06 20:02:02,535 DEBUG   SenderThread:2148567 [sender.py:send():382] send: history
2024-08-06 20:02:02,535 DEBUG   SenderThread:2148567 [sender.py:send_request():409] send_request: summary_record
2024-08-06 20:02:02,536 INFO    SenderThread:2148567 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-06 20:02:02,764 INFO    Thread-12 :2148567 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_200158-zo7etxt6/files/wandb-summary.json
2024-08-06 20:02:03,765 INFO    Thread-12 :2148567 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_200158-zo7etxt6/files/output.log
2024-08-06 20:02:04,063 DEBUG   HandlerThread:2148567 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 20:02:04,063 DEBUG   HandlerThread:2148567 [handler.py:handle_request():146] handle_request: partial_history
2024-08-06 20:02:04,064 DEBUG   SenderThread:2148567 [sender.py:send():382] send: history
2024-08-06 20:02:04,064 DEBUG   SenderThread:2148567 [sender.py:send_request():409] send_request: summary_record
2024-08-06 20:02:04,064 INFO    SenderThread:2148567 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-06 20:02:04,701 ERROR   wandb-upload_0:2148567 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/zo7etxt6/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T120200Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=8770319683b94cf3b7889fb807301df06cdc0e102e6e00518805bd314884a1aa: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/zo7etxt6/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T120200Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=8770319683b94cf3b7889fb807301df06cdc0e102e6e00518805bd314884a1aa
2024-08-06 20:02:04,701 ERROR   wandb-upload_0:2148567 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5174', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 20:02:04,701 ERROR   wandb-upload_0:2148567 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/zo7etxt6/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/zo7etxt6/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E922020E829ADB</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 20:02:04,765 INFO    Thread-12 :2148567 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_200158-zo7etxt6/files/wandb-summary.json
2024-08-06 20:02:04,765 INFO    Thread-12 :2148567 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_200158-zo7etxt6/files/output.log
2024-08-06 20:02:04,939 ERROR   wandb-upload_1:2148567 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/zo7etxt6/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T120200Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=50a85cf9e52b42c40f2c8d915d09a31c5e68476df98b8fa9ee6a4cd2ac8adce4: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/zo7etxt6/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T120200Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=50a85cf9e52b42c40f2c8d915d09a31c5e68476df98b8fa9ee6a4cd2ac8adce4
2024-08-06 20:02:04,939 ERROR   wandb-upload_1:2148567 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 20:02:04,939 ERROR   wandb-upload_1:2148567 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/zo7etxt6/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/zo7etxt6/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E922021CF6E427</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 20:02:05,504 DEBUG   HandlerThread:2148567 [handler.py:handle_request():146] handle_request: partial_history
2024-08-06 20:02:05,505 DEBUG   SenderThread:2148567 [sender.py:send():382] send: history
2024-08-06 20:02:05,505 DEBUG   SenderThread:2148567 [sender.py:send_request():409] send_request: summary_record
2024-08-06 20:02:05,505 INFO    SenderThread:2148567 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-06 20:02:05,765 INFO    Thread-12 :2148567 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_200158-zo7etxt6/files/wandb-summary.json
2024-08-06 20:02:05,765 INFO    Thread-12 :2148567 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_200158-zo7etxt6/files/output.log
2024-08-06 20:02:06,765 INFO    Thread-12 :2148567 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_200158-zo7etxt6/files/output.log
2024-08-06 20:02:07,089 DEBUG   HandlerThread:2148567 [handler.py:handle_request():146] handle_request: partial_history
2024-08-06 20:02:07,090 DEBUG   SenderThread:2148567 [sender.py:send():382] send: history
2024-08-06 20:02:07,090 DEBUG   SenderThread:2148567 [sender.py:send_request():409] send_request: summary_record
2024-08-06 20:02:07,091 INFO    SenderThread:2148567 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-06 20:02:07,765 INFO    Thread-12 :2148567 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_200158-zo7etxt6/files/wandb-summary.json
2024-08-06 20:02:07,765 INFO    Thread-12 :2148567 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_200158-zo7etxt6/files/output.log
2024-08-06 20:02:08,630 DEBUG   HandlerThread:2148567 [handler.py:handle_request():146] handle_request: partial_history
2024-08-06 20:02:08,631 DEBUG   SenderThread:2148567 [sender.py:send():382] send: history
2024-08-06 20:02:08,631 DEBUG   SenderThread:2148567 [sender.py:send_request():409] send_request: summary_record
2024-08-06 20:02:08,631 INFO    SenderThread:2148567 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-06 20:02:08,765 INFO    Thread-12 :2148567 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_200158-zo7etxt6/files/wandb-summary.json
2024-08-06 20:02:08,766 INFO    Thread-12 :2148567 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_200158-zo7etxt6/files/output.log
2024-08-06 20:02:08,994 ERROR   wandb-upload_0:2148567 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/zo7etxt6/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T120200Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=8770319683b94cf3b7889fb807301df06cdc0e102e6e00518805bd314884a1aa: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/zo7etxt6/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T120200Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=8770319683b94cf3b7889fb807301df06cdc0e102e6e00518805bd314884a1aa
2024-08-06 20:02:08,994 ERROR   wandb-upload_0:2148567 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5174', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 20:02:08,994 ERROR   wandb-upload_0:2148567 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/zo7etxt6/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/zo7etxt6/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E922030A1D74F0</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 20:02:09,632 DEBUG   HandlerThread:2148567 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 20:02:09,766 INFO    Thread-12 :2148567 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_200158-zo7etxt6/files/output.log
2024-08-06 20:02:09,994 ERROR   wandb-upload_1:2148567 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/zo7etxt6/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T120200Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=50a85cf9e52b42c40f2c8d915d09a31c5e68476df98b8fa9ee6a4cd2ac8adce4: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/zo7etxt6/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T120200Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=50a85cf9e52b42c40f2c8d915d09a31c5e68476df98b8fa9ee6a4cd2ac8adce4
2024-08-06 20:02:09,994 ERROR   wandb-upload_1:2148567 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 20:02:09,994 ERROR   wandb-upload_1:2148567 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/zo7etxt6/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/zo7etxt6/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E922034376946B</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 20:02:10,162 DEBUG   HandlerThread:2148567 [handler.py:handle_request():146] handle_request: partial_history
2024-08-06 20:02:10,163 DEBUG   SenderThread:2148567 [sender.py:send():382] send: history
2024-08-06 20:02:10,163 DEBUG   SenderThread:2148567 [sender.py:send_request():409] send_request: summary_record
2024-08-06 20:02:10,163 INFO    SenderThread:2148567 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-06 20:02:10,766 INFO    Thread-12 :2148567 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_200158-zo7etxt6/files/wandb-summary.json
2024-08-06 20:02:10,766 INFO    Thread-12 :2148567 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_200158-zo7etxt6/files/output.log
2024-08-06 20:02:11,676 DEBUG   HandlerThread:2148567 [handler.py:handle_request():146] handle_request: partial_history
2024-08-06 20:02:11,677 DEBUG   SenderThread:2148567 [sender.py:send():382] send: history
2024-08-06 20:02:11,677 DEBUG   SenderThread:2148567 [sender.py:send_request():409] send_request: summary_record
2024-08-06 20:02:11,677 INFO    SenderThread:2148567 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-06 20:02:11,766 INFO    Thread-12 :2148567 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_200158-zo7etxt6/files/wandb-summary.json
2024-08-06 20:02:11,766 INFO    Thread-12 :2148567 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_200158-zo7etxt6/files/output.log
2024-08-06 20:02:12,766 INFO    Thread-12 :2148567 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_200158-zo7etxt6/files/output.log
2024-08-06 20:02:13,058 DEBUG   SenderThread:2148567 [sender.py:send():382] send: exit
2024-08-06 20:02:13,058 INFO    SenderThread:2148567 [sender.py:send_exit():589] handling exit code: 255
2024-08-06 20:02:13,058 INFO    SenderThread:2148567 [sender.py:send_exit():591] handling runtime: 14
2024-08-06 20:02:13,058 INFO    SenderThread:2148567 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-06 20:02:13,058 INFO    SenderThread:2148567 [sender.py:send_exit():597] send defer
2024-08-06 20:02:13,058 DEBUG   HandlerThread:2148567 [handler.py:handle_request():146] handle_request: defer
2024-08-06 20:02:13,058 INFO    HandlerThread:2148567 [handler.py:handle_request_defer():172] handle defer: 0
2024-08-06 20:02:13,058 DEBUG   SenderThread:2148567 [sender.py:send_request():409] send_request: defer
2024-08-06 20:02:13,058 INFO    SenderThread:2148567 [sender.py:send_request_defer():613] handle sender defer: 0
2024-08-06 20:02:13,058 INFO    SenderThread:2148567 [sender.py:transition_state():617] send defer: 1
2024-08-06 20:02:13,058 DEBUG   HandlerThread:2148567 [handler.py:handle_request():146] handle_request: defer
2024-08-06 20:02:13,058 INFO    HandlerThread:2148567 [handler.py:handle_request_defer():172] handle defer: 1
2024-08-06 20:02:13,058 DEBUG   SenderThread:2148567 [sender.py:send_request():409] send_request: defer
2024-08-06 20:02:13,058 INFO    SenderThread:2148567 [sender.py:send_request_defer():613] handle sender defer: 1
2024-08-06 20:02:13,058 INFO    SenderThread:2148567 [sender.py:transition_state():617] send defer: 2
2024-08-06 20:02:13,058 DEBUG   HandlerThread:2148567 [handler.py:handle_request():146] handle_request: defer
2024-08-06 20:02:13,058 INFO    HandlerThread:2148567 [handler.py:handle_request_defer():172] handle defer: 2
2024-08-06 20:02:13,058 INFO    HandlerThread:2148567 [system_monitor.py:finish():203] Stopping system monitor
2024-08-06 20:02:13,058 INFO    HandlerThread:2148567 [interfaces.py:finish():202] Joined cpu monitor
2024-08-06 20:02:13,059 INFO    HandlerThread:2148567 [interfaces.py:finish():202] Joined disk monitor
2024-08-06 20:02:13,059 DEBUG   SystemMonitor:2148567 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-08-06 20:02:13,059 DEBUG   SystemMonitor:2148567 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-08-06 20:02:13,059 DEBUG   SystemMonitor:2148567 [system_monitor.py:_start():183] Publishing last batch of metrics
